### PR TITLE
tree-wide: minor formatting inconsistency cleanups

### PR DESCRIPTION
**Pull request by vcaputo**

### CODE REVIEW

**Comment by vcaputo** [comment]
some more purely mechanical cleanups

File: src/journal/journal-verify.c [hunk 1]
```diff
@@ -97,20 +97,20 @@ static void flush_progress(void) {
         fflush(stdout);
 }
 
-#define debug(_offset, _fmt, ...) do{                                   \
+#define debug(_offset, _fmt, ...) do {                                   \
```
Comment (keszybz) [inline src/journal/journal-verify.c]: That's ugly :(

**Comment by keszybz** [comment]
Can you fix `src/journal/journal-verify.c` please?

**Comment by vcaputo** [comment]
Of course, I completely missed that shifting in review of what my one-liner did, thanks for the review and catching it.

**Comment by vcaputo** [comment]
@keszybz updated!

**Pull request merged**